traceloader.py

```diff
@@ -0,0 +1,137 @@
+import json
+import os
+
+import numpy as np
+
+from utils import tools
+import scipy.stats as stats
+
+def gauss(mu, sigma, number):
+    '''
+    Return `number`-length truncated Gaussian sequence.
+
+    Range: [mu - 3 sigma, mu + 3 sigma]
+
+    Args:
+        mu (int/float): mean
+        sigma (int/float): variance       
+        number (int): the length of truncated Gaussian sequence
+
+    Returns:
+        np.ndarray: a number-length truncated Gaussian sequence
+    '''
+    lower, upper = -3, 3
+    X = stats.truncnorm(lower, upper, loc=mu, scale=sigma)
+    return X.rvs(size=number)
+
+class traceloader:
+    'load the information of traces'
+    def __init__(self, trace_list:list, trace_folder:str, raw_save=0, trace_info_save=0):
+        '''
+        initialize the traceloader
+
+        Args:
+        trace_list: information of traces to load
+        trace_folder: path to save the traces
+        binsave: whether save the raw_data in .npy or not
+        jsonsave: whether save the raw_data in .json or not
```

Comment:
Regenerate the documentation. You specify type in argument, so that documentation will have type information.

Reply:
Please use the vsc's extension `Python Docstring Generator` to generate a new argument. If you specify the type, it will take it. If you don't, it will let you replace `<type>` yourself.